src/app/content/Content.tsx

```diff
@@ -1,57 +1,85 @@
  */
 export function Content({ term }: ContentProps): JSX.Element {
   const [searchParams] = useSearchParams();
   const { data, loading } = useGetCourse({ term, pid: searchParams.get('pid') || '' });
+  console.log(data);
 
   return (
-    <Box width={['container.md', 'container.lg', 'container.xl']} bg="white" p={4} zIndex={60}>
-      <Flex
-        justifyItems="center"
-        alignItems={{ base: 'start', sm: 'center' }}
-        direction={{ base: 'column', sm: 'row' }}
-      >
-        <Heading mr="5" size="2xl" as="h2" whiteSpace="pre" color="black">{`${data?.subject || ''} ${
-          data?.code || ''
-        }`}</Heading>
-        {!loading && data && (
-          <Heading size="lg" as="h3" color="gray">
-            {data.title}
-          </Heading>
-        )}
-      </Flex>
-      <Skeleton isLoaded={!loading}>
-        {data && (
-          <>
-            <CourseInfo
-              subject={data.subject}
-              code={data.code}
-              title={data.title}
-              description={data.description || ''}
-              credits={data.credits}
-              hours={data.hoursCatalog}
-            />
-            <SectionsContainer term={term} subject={data?.subject} code={data?.code} />
-          </>
-        )}
-      </Skeleton>
-    </Box>
+    <Flex width={['container.md', 'container.lg', 'container.xl']} flexDirection="column" minH="100%">
+      <Box bg="white" p={4} zIndex={60}>
+        <Flex
+          justifyItems="center"
+          alignItems={{ base: 'start', sm: 'center' }}
+          direction={{ base: 'column', sm: 'row' }}
+        >
+          <Heading mr="5" size="2xl" as="h2" whiteSpace="pre" color="black">{`${data?.subject || ''} ${
+            data?.code || ''
+          }`}</Heading>
+          {!loading && data && (
+            <Heading size="lg" as="h3" color="gray">
+              {data.title}
+            </Heading>
+          )}
+        </Flex>
+        <Skeleton isLoaded={!loading}>
+          {data && (
+            <>
+              <CourseInfo
+                subject={data.subject}
+                code={data.code}
+                title={data.title}
+                description={data.description || ''}
+                credits={data.credits}
+                hours={data.hoursCatalog}
+              />
+              <SectionsContainer term={term} subject={data?.subject} code={data?.code} />
+            </>
+          )}
+        </Skeleton>
+      </Box>
+      <Spacer />
+      <Center>
+        <Text as="span" fontWeight="bold" fontSize={12}>
+          Sources:
+          <Text as="span" color="blue.500" decoration="underline" fontWeight="light">
+            <Text
+              as="a"
+              href={`https://www.uvic.ca/calendar/undergrad/index.php#/courses/${data?.pid}`}
+              target="_blank"
+              mx="3"
+              _hover={{ color: 'blue' }}
+            >
+              {`UVic Undergraduate Calendar ${new Date().getFullYear()}`}
```

Comment:
Doesn't this need to be tied to the term?

Reply:
The link?

Reply:
The year. If you select Fall right now it's technically wrong.

Reply:
Ah yes. in that case... the content is static across terms, maybe should just remove the year

Reply:
Yeah let's go with that.